# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/basic/bug1601074.js
# Commit: e4c1a6acc2b1
# Full Hash: e4c1a6acc2b152f69aefa80f46b8c14d999cc385
# Author: Chris Fallin <cfallin@mozilla.com>
# Date: 2019-12-05 04:11:08
# Regressor Bug: 1598347
# File Overlap Count: 2
# Description:
#   Bug 1601074: Fix preliminary-objects-set issue wrt NEWOBJECT_WITHGROUP. r=iain
#   
#   The recent addition of the JSOP_NEWOBJECT_WITHGROUP opcode for bug
#   1598347 (itself a regression fix for 1580246) has led to an issue when
#   more than a certain number of array elements with the same group are
# ==============================================================================

diff -r d8b0a5b0a5af -r e4c1a6acc2b1 js/src/jit-test/tests/basic/bug1601074.js
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/js/src/jit-test/tests/basic/bug1601074.js	Wed Dec 04 00:47:39 2019 +0000
@@ -0,0 +1,13 @@
+for (let y of [
+      { x: 1 },
+      { x: 2 },
+      { x: 3 },
+      { x: 4 },
+      { x: 5 },
+      { x: 6 },
+      { x: 7 },
+      { x: 8 },
+      { x: 9 },
+      { x: 10 },
+      { x: 11 },
+]) {}